nncf/tensorflow/graph/transformations/commands.py

```diff
@@ -47,7 +57,31 @@ def __eq__(self, other: 'TFLayerPoint') -> bool:
     def __str__(self) -> str:
         return super().__str__() + ' ' + self.layer_name
 
-
+    def get_state(self) -> Dict[str, Any]:
```

Comment:
Could you please add the `:return:` section here and below (for all classes)?

Reply:
Done